src/main/resources/plugin.yml

```diff
@@ -9,10 +9,22 @@ commands:
       usage: /<command> [off|on|reload]
       permission: rwm.redworldinfo.use
       permission-message: You don't have the permission <permission>
+   redseedinfo:
+      aliases: [seed]
+      description: Command description.
```

Comment:
Even though the command description is not really necessary it shouldn't hurt, if there would be a correct text.

Reply:
In the new commit the description is now more detailed